### PR TITLE
Added Kari's answers to core group questions

### PR DESCRIPTION
**Pull request by kariljordan**
@gvwilson Added Kari's answers to "who else do you need".

### CODE REVIEW

File: _exercises/02-05-b-building-your-core-group.md [hunk 1]
```diff
@@ -23,3 +23,17 @@ title: "Building Your Core Group"
 
 6.  Who has these?
     [____________________]
+# Kari L. Jordan
+
+1.  Knowing your personality, what other kinds of personalities would be most useful in the core group?
+    Critical thinkers, those with a mind for marketing, and those who enjoy philanthropy.
```
Comment (gvwilson) [inline _exercises/02-05-b-building-your-core-group.md]: You don't enjoy philanthropy??

File: _exercises/02-05-b-building-your-core-group.md [hunk 2]
```diff
@@ -23,3 +23,17 @@ title: "Building Your Core Group"
...
+# Kari L. Jordan
+
+1.  Knowing your personality, what other kinds of personalities would be most useful in the core group?
+    Critical thinkers, those with a mind for marketing, and those who enjoy philanthropy. 
+2.  What special knowledge do you have?
+    Knowledge related to diverse populations and STEM-Education/Careers.
+3.  What special knowledge or skills do you lack that the core group needs?
+    Fundraising
+4.  Who has this knowledge or these skills?
+    That remains to be seen. People I know who are good at fundraising already have jobs.
```
Comment (gvwilson) [inline _exercises/02-05-b-building-your-core-group.md]: Does that mean they won't answer questions or mentor you?

File: _exercises/02-05-b-building-your-core-group.md (hunk 1 above)
Comment (kariljordan) [inline _exercises/02-05-b-building-your-core-group.md]: @gvwilson I don't feel comfortable asking people for money. I know there is more to philanthropy than that, but that's what it sort of boils down to. I enjoy "telling the story" and sharing experiences, but when it comes to the "big ask" I'm not really comfortable doing that. I have experience with that from when I worked for the YMCA. I hated it!

File: _exercises/02-05-b-building-your-core-group.md (hunk 2 above)
Comment (kariljordan) [inline _exercises/02-05-b-building-your-core-group.md]: @gvwilson I never thought about mentorship! I can ask for tips and for them to help me grow in this area. Thanks Greg.

File: _exercises/02-05-b-building-your-core-group.md (hunk 1 above)
Comment (raynamharris) [inline _exercises/02-05-b-building-your-core-group.md]: I'm with you there @kariljordan. I don't like asking for money, and I never enjoyed the sales aspects of previous jobs.

Comment (gvwilson) [inline _exercises/02-05-b-building-your-core-group.md]: You're not "asking for money" - you're "offering people who might not be able to contribute their own time help the project by giving other people the means to do so".

Comment (apawlik) [inline _exercises/02-05-b-building-your-core-group.md]: @gvwilson The phrasing you propose is useful. I made a comment in https://github.com/gvwilson/bpco/pull/17 And now reading @kariljordan and @raynamharris comments I think I (we?) could discuss this more: money in communities/initiatives for making the world better. The number of times I witnessed the ironic smile (at best) and a comment "Ah, right-to. So it IS actually about money and not the idea? Oh, well, you can buy food for ideas."

I think there is a lot of attitude that organisations that are for a good cause should basically run on air (and 100% volunteer effort). If anyone is paid, or anything is purchased, you feel like you need to apologise and excuse it.

Comment (kariljordan) [inline _exercises/02-05-b-building-your-core-group.md]: Wow @gvwilson. I love the way you put that. WOW! And yes @apawlik, people don't understand that you do need money to run a non-profit...

**Pull request merged**